packages/server/src/database/service.ts

```diff
@@ -82,6 +82,10 @@ export class DatabaseService extends Service {
   }
 
   async destroy() {
+    // This delay is necessary to assure that all queries to the database are done before closing the connection
+    const delay = 1000
```

Comment:
What guarantee do you have that after a second all queries will have finished?

Reply:
None this is pretty much a hack. But if we have queries that take over a second then we have a problem elsewhere anyways. I couldn't find a "waitForOngoingOperations()" or something function in knex so that's the best solution that works.